azure-pipelines.yml

```diff
@@ -51,3 +51,47 @@ jobs:
 
   steps:
   - template: ci/azure-steps.yml
+
+- job: cygwin
```

Comment:
Might make sense to name this something like `cygwin gccx64ninja` for consistency with the other jobs